### PR TITLE
boards: rv-virt: Fix stack corruption for rv64 configs

### PR DESCRIPTION
**Pull request by masayuki2009**
## Summary

- This commit fixes stack corruption for rv64 configs

## Impact

- None

## Testing

- Tested with qemu-6.2

### CODE REVIEW

**Review by pkarashchenko — APPROVED** [review]

File: boards/risc-v/qemu-rv/rv-virt/configs/nsh64/defconfig [hunk 1]
```diff
@@ -47,6 +47,7 @@ CONFIG_DEV_ZERO=y
 CONFIG_FS_PROCFS=y
 CONFIG_IDLETHREAD_STACKSIZE=2048
 CONFIG_INIT_ENTRYPOINT="nsh_main"
```
Comment (no1wudi) [inline boards/risc-v/qemu-rv/rv-virt/configs/nsh64/defconfig:49]: It's better to increase stack size for nsh also, like maix-bit.

File: boards/risc-v/qemu-rv/rv-virt/configs/nsh64/defconfig [hunk 2]
```diff
@@ -65,6 +66,7 @@ CONFIG_STACK_COLORATION=y
 CONFIG_START_MONTH=12
 CONFIG_START_YEAR=2021
 CONFIG_SYSTEM_NSH=y
+CONFIG_SYSTEM_NSH_STACKSIZE=3072
```
Comment (no1wudi) [inline boards/risc-v/qemu-rv/rv-virt/configs/nsh64/defconfig:69]: Please change to CONFIG_INIT_STACKSIZE=3072

Comment (masayuki2009) [reply boards/risc-v/qemu-rv/rv-virt/configs/nsh64/defconfig:69]: @no1wudi 
I've already changed the CONFIG_INIT_STACKSIZE.

Comment (no1wudi) [reply boards/risc-v/qemu-rv/rv-virt/configs/nsh64/defconfig:69]: CONFIG_ARCH_INTERRUPTSTACK seems restored to 2048

Comment (masayuki2009) [reply boards/risc-v/qemu-rv/rv-virt/configs/nsh64/defconfig:69]: @no1wudi 
As long as I tested, 2048 is enough for CONFIG_ARCH_INTERRUPTSTACK.

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**